packages/fe-fpm-writer/src/building-block/index.ts

```diff
@@ -25,6 +26,16 @@ interface MetadataPath {
     metaPath: string;
 }
 
+/**
+ * Gets manifest path.
+ *
+ * @param {string} basePath the base path
+ * @returns {Manifest | undefined} path to manifest file
+ */
+function getManifestPath(basePath: string): string {
+    return join(basePath, 'webapp/manifest.json');
```

Comment:
```suggestion
    return join(await getWebappPath(basePath), FileName.Manifest);
```
Better not to hardcode webapp path

Reply:
@devinea 
applied correct, but one of problem is that API methods became asynchronous -> commit ccfa3e095b334bc5f70df2048b5bb2caaaf830f9